tests/vmcheck/test-override-kernel.sh

```diff
@@ -33,14 +33,14 @@ vm_cmd 'curl -sS -L -O https://dl.fedoraproject.org/pub/fedora/linux/releases/27
 current=$(vm_get_booted_csum)
 vm_cmd rpm-ostree db list "${current}" > current-dblist.txt
 assert_not_file_has_content current-dblist.txt 'kernel-4.13.9-300.fc27'
-grep -E '^ kernel-' current-dblist.txt  | sed -e 's,^ *,,' > current-kernel.txt
+grep -E '^ kernel-4' current-dblist.txt  | sed -e 's,^ *,,' > orig-kernel.txt
+assert_streq "$(wc -l orig-kernel.txt | cut -f 1 -d ' ')" "1"
```

Comment:
FWIW, `wc -l < orig-kernel.txt` avoids the cut. ;)
It's funny, because that's exactly the reason I used `cat`, but I somehow missed the obvious `<`!

Reply:
Oh man I can't believe I dinged your for doing it then went and did it 5 seconds later...

Reply:
Wait I didn't quite do it!  OK I get it now :smile: